README.md

```diff
@@ -46,12 +46,14 @@ python application.py runserver
 
 ### Using the API locally
 
-Calls to the API require a valid bearer token. Tokens to be accepted can be set using the AUTH_TOKENS environment variable, e.g.:
+** BEARER TOKEN AUTH IS NOT YET IMPLEMENTED, BUT SHOULD BE VERY SOON! **
```

Comment:
Yep we could. This was a kind-of reminder that it needs to be done.  But probably shouldn't be in the documentation.